deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -171,13 +178,6 @@ spec:
           - deployments/finalizers
           verbs:
           - update
-        - apiGroups:
-            - ""
```

Comment:
@PratikMahajan - this is what I was talking about

Reply:
just saw that they're just being rearranged.